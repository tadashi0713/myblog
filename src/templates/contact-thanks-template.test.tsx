import React from 'react';
import renderer from 'react-test-renderer';
import ContactThanksTemplate from './contact-thanks-template';

describe('ContactThanksTemplate', () => {
  const props = {
    data: {
      site: {
        siteMetadata: {
          title: 'test',
          subtitle: 'test'
        }
      }
    }
  };

  it('renders correctly', () => {
    const tree = renderer.create(<ContactThanksTemplate {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
