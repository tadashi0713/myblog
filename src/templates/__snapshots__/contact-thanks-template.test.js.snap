// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactThanksTemplate renders correctly 1`] = `
<div
  className="layout"
>
  <StaticQuery
    render={[Function]}
  />
  <div
    className="page"
  >
    <div
      className="page__inner"
    >
      <h1
        className="page__title"
      >
        Thank you
      </h1>
      <div
        className="page__body"
      >
        <p>
          Thank you for contact
        </p>
      </div>
    </div>
  </div>
</div>
`;
